src/gui/tray/CustomTextButton.qml

```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.3
+import Style 1.0
+
+Item {
```

Comment:
This should ideally inherit from QtQuick.Controls.AbstractButton so that accessibility and keyboard navigation is provided built-in

Reply:
@CarlSchwan Thanks, but, the reason I did not use the **AbstractButton** is I could not find a way to customize its **MouseArea**. I need it to wrap the **Label** but, with **AbstractButton** it wraps the entire control, including the invisible area, resulting in mouse hover and click to get triggered without having a cursor on top of a label but, close to it.

Any ideas on how to resize the built-in MouseArea of the AbstructButton? I can see that I can only customize the `contentItem` and `background` (no mention of `MouseArea` in the docs).

P.S.: No, I can not set the height of a button to the height of a `Label` because the, I will have to always write extra code t vertically center it with another button that has a blue background, so, I'd prefer heights to be the same.

Now, I could still try and put `MouseArea` into the `AbstractButton`, but, won't that be wrong? The built-in `MouseArea` would still be there and it would be processing cursor events.

Reply:
Yeah padding/inset might help, or alternatively just use a Label as top element with a MouseArea inside like in https://invent.kde.org/frameworks/kirigami/-/blob/master/src/controls/LinkButton.qml

Reply:
thing is we may never really write a real Qt quick controls theme so going with simplicity is most probably the right choice